### PR TITLE
fix: added all contributors in footer

### PR DESCRIPTION
**Pull request by indraasura**
Fixes 

**Checklist:**

- [ ] I have squashed my commits
- [x] My PR follows the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream development/master branch.

Changes:
Added remaining contributors in the footer

**Screenshots for the change**
Screenshot_5

### CODE REVIEW

**Review by shashank-sharma — CHANGES_REQUESTED** [review]
Few changes:
1. Fix `alt=Harshit` attribute with their respective names
2. Make sure that every contributor names is mentioned including you I guess
3. Cross check link and profile picture which were placed

**Comment by mooocer** [comment]
@indraasura you also didn't link the PR to the issue. This is important to automatically close the issue.

**Comment by indraasura** [comment]
> @indraasura you also didn't link the PR to the issue. This is important to automatically close the issue.

Did that now

**Comment by indraasura** [comment]
> Few changes:
> 
> 1. Fix `alt=Harshit` attribute with their respective names
> 2. Make sure that every contributor names is mentioned including you I guess
> 3. Cross check link and profile picture which were placed

Sure, will do it by today

File: index.html [hunk 1]
```diff
@@ -245,24 +245,49 @@ <h3 class="h2">Contact Us</h3>
...
+                            </a>
+                        </div>
+                        <div class="col-sm-1">
+                            <a href="https://github.com/vbhv98">
+                                <img src="https://avatars0.githubusercontent.com/u/22888661?s=460&v=4" class="rounded-circle person" alt="Hariom Verma" width="40" height="40">
+                            </a>
+                        </div>
+                        <div class="col-sm-1">
+                            <a href="https://github.com/harry-hov">
+                                <img src="https://avatars3.githubusercontent.com/u/37576387?s=460&v=4" class="rounded-circle person" alt="Vaibhav Bhawalkar" width="40" height="40">
```
Comment (mooocer) [inline index.html]: Wrong alt attribute, mistakenly interchanged

**Comment by mooocer** [comment]
Correct the names and squash the commits. Please complete it before the events one.

**Review by mooocer — CHANGES_REQUESTED** [review]
Small change left.

**Pull request closed without merging**